src/main/java/org/folio/qm/service/impl/MarcRecordsServiceImpl.java

```diff
@@ -71,6 +71,7 @@ public QuickMarc findByInstanceId(UUID instanceId) {
   @Override
   public void updateById(UUID instanceId, QuickMarc quickMarc) {
```

Comment:
And what about creating a new record? We should validate it also.

Reply:
Yes, it is currently working when we use `hibernate-validator`